tests/AppTest.php

```diff
@@ -219,8 +218,10 @@ public function testHttpFlowEventNotInstanceOfEventInterface()
         $response = $app->run($request, $response);
     }
 
-    public function testRouteInfoNotInstanceOfRouteInfoInterface()
+    public function testDispatcherNotInstanceOfPennyDispatcher()
     {
+        $this->setExpectedException('RuntimeException', 'Dispatcher must be an instance of Penny\Dispatcher class');
```

Comment:
This use case for me is good.. The dispatcher is more flexible.. Is the App that should manage its response..